src/types/inc/utils.hpp

```diff
@@ -24,11 +36,26 @@ namespace Microsoft::Console::Utils
     std::string ColorToHexString(const COLORREF color);
     COLORREF ColorFromHexString(const std::string wstr);
 
-    void InitializeCampbellColorTable(gsl::span<COLORREF>& table);
-    void InitializeCampbellColorTableForConhost(gsl::span<COLORREF>& table);
-    void SwapANSIColorOrderForConhost(gsl::span<COLORREF>& table);
-    void Initialize256ColorTable(gsl::span<COLORREF>& table);
-    void SetColorTableAlpha(gsl::span<COLORREF>& table, const BYTE newAlpha);
+    void InitializeCampbellColorTable(const gsl::span<COLORREF> table);
+    void InitializeCampbellColorTableForConhost(const gsl::span<COLORREF> table);
+    void SwapANSIColorOrderForConhost(const gsl::span<COLORREF> table);
+    void Initialize256ColorTable(const gsl::span<COLORREF> table);
+
+    // Function Description:
+    // - Fill the alpha byte of the colors in a given color table with the given value.
+    // Arguments:
+    // - table: a color table
+    // - newAlpha: the new value to use as the alpha for all the entries in that table.
+    // Return Value:
+    // - <none>
+    constexpr void SetColorTableAlpha(const gsl::span<COLORREF> table, const BYTE newAlpha) noexcept

```

Comment:
`static`?

Reply:
Uh, it's not a class. It's just sitting in a namespace. So it's already static. Also constexpr.